### ISSUE TITLE
Add committees as keywords/tags that people can subscribe to / follow

### ISSUE THREAD

**Issue by jayapapaya** [d-cent/democracyos#45]
As a User, i need a button in context to subscribe to a keyword.

**Comment by jaakkokorhonen** [comment]
Is committee an actor? Maybe it should be a user/group?

**Comment by jaakkokorhonen** [comment]
Admins can have confidential documents that are then published. 
Users can add keywords to their profiles.

**Comment by jaakkokorhonen** [comment]
@gvilarino any ideas on how to inject keywords to user profile settings, so that keywords can be shown as subscribeable buttons in comments or documents?

**Comment by gvilarino** [comment]
@jaakkokorhonen I'm not sure I'm understanding what you mean with your question. Can you give an example?

btw: admins CAN have confidential documents Those are what we call drafts: unpublished laws that all admins can see. The admin doesn't display what `law`s are drafts on the list, but that could be sorted out easily.

**Comment by jaakkokorhonen** [comment]
As user, when i'm reading a article, i want to see the keywords for the article and subscribe to them. ideally, there could be a http post api to receive user's new keywords.

**Comment by gvilarino** [comment]
I agree, but that wasn't specified in #5 or #11. This could be included in a future milestone.
